datalake/src/main/java/com/sequenceiq/datalake/service/validation/converter/CredentialToCloudCredentialConverter.java

```diff
@@ -0,0 +1,25 @@
+package com.sequenceiq.datalake.service.validation.converter;
+
+import static org.apache.commons.lang3.StringUtils.isEmpty;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.cloud.model.CloudCredential;
+import com.sequenceiq.cloudbreak.common.json.Json;
+import com.sequenceiq.datalake.entity.Credential;
+
+@Component
+public class CredentialToCloudCredentialConverter {
+
+    public CloudCredential convert(Credential credential) {
+        if (credential == null) {
```

Comment:
use `getIfNotNull` like in the other converters

Reply:
ok